src/actions/index.js

```diff
@@ -0,0 +1,15 @@
+
+export const ADD_TODO = 'ADD_TODO';
+
```

Comment:
To keep actions clear, we can just pass payload here.

```
export function addTodo(payload) {
  return {
    type: ADD_TODO,
    payload
  };
} 
```

The `payload` will contain your todo item, and you can generate ID in the saga (where you can also destructure payload to access todo data).